data-prepper-plugins/opensearch/src/main/resources/logs-policy-no-ism-template.json

```
@@ -0,0 +1,19 @@
+{
+  "policy": {
+    "description": "Managing logs for log analytics",
+    "default_state": "current_write_index",
+    "states": [
+      {
+        "name": "current_write_index",
+        "actions": [
+          {
+            "rollover": {
+              "min_size": "50gb",
```

This might be too large as a default.
```suggestion
              "min_size": "10gb",
```

---

I aligned with the default for traces, which is also 50 GB

https://github.com/opensearch-project/data-prepper/blob/efda3e77d5bbac7f92de11ddf05620529bbeec3a/data-prepper-plugins/opensearch/src/main/resources/raw-span-policy-no-ism-template.json#L11